### PR TITLE
Adds Boom compatibility

### LINKED ISSUE #49: Always has 500 error even if ctx.status is not 500
I'm using this in combination with https://github.com/niftylettuce/koa-better-error-handler.

On the console it says the error code is 500, but in reality the error code / response code was non-500.  What can we do here to fix this?

cc @coderhaoxin

### PR DESCRIPTION
**Pull request by OmgImAlexis**
Closes #49

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ function dev (opts) {
 function log (ctx, start, len, err, event) {
   // get the status code of the response
   const status = err
-    ? (err.status || 500)
+    ? (err.isBoom ? err.output.statusCode : err.status || 500)
```
Comment (coderhaoxin) [inline index.js]: @OmgImAlexis can you add a test please :)

Comment (OmgImAlexis) [reply index.js]: Should I add boom as a dep for the test?

Comment (coderhaoxin) [reply index.js]: > Should I add boom as a dep for the test?

You can add `boom` as a devDep :)

**Comment by niftylettuce** [comment]
looks good, ping me when a release is cut to npm! 🙇

**Comment by coderhaoxin** [comment]
`3.1.0` published :)

**Pull request merged**